riscv-executor/src/lib.rs

```diff
@@ -645,59 +904,202 @@ struct Executor<'a, 'b, F: FieldElement> {
     label_map: HashMap<&'a str, Elem<F>>,
     inputs: &'b Callback<'b, F>,
     bootloader_inputs: Vec<Elem<F>>,
-    _stdout: io::Stdout,
+    fixed: Arc<Vec<(String, VariablySizedColumn<F>)>>,
+    // program columns: maps fixed cols to respective witness cols
```

Comment:
maps fixed to witness? wdym?

Reply:
maps the "rom" `p_foo` fixed columns to their respective `foo` witness column